core/templates/dev/head/css/oppia.css

```diff
@@ -2322,10 +2335,31 @@ div#ng-curtain {
   margin-bottom: 10px;
 }
 
+.oppia-notifications-tray {
+  margin-left: -8px;
+}
+
 .oppia-notifications-dashboard-card {
   padding: 15px 40px;
 }
 
+.oppia-notification-content {
+  display: inline-block;
+}
+
+.oppia-notification-last-updated {
+  float: right;
+}
+
+@media all and (max-width: 500px) {
```

Comment:
Just checking: why 'all' here but not in line 117? (Not sure which is better, but the inconsistency seems odd.)